x-pack/plugins/apm/public/components/app/RumDashboard/hooks/useHasRumData.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useFetcher } from '../../../../hooks/use_fetcher';
+
+export function useHasRumData() {
+  const { data } = useFetcher((callApmApi) => {
```

Comment:
I'm ++ to @smith - don't see a downside to simply:

```suggestion
  return useFetcher((callApmApi) => {
```

Then in your above implementation just

```typescript
const { data: rumData } = useHasRumData();
```